CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ check_ipo_supported(RESULT IPO_SUPPORTED OUTPUT error)
 if(IPO_SUPPORTED)
     set(CMAKE_INTERPROCEDURAL_OPTIMIZATION
         $<$<CONFIG:DEBUG>:OFF>
+        $<$<CONFIG:TEST>:ON>
```

Comment:
is TEST an actual config or is this an artefact?

Reply:
oops - artifact of trying to make a whole new config before deciding to just use relwithdebinfo instead